### ISSUE TITLE
Costing Specification to Add Non RSMeans components to the database and incorporate into BTAP

### ISSUE THREAD

**Issue by mlubun** [canmet-energy/btap_tasks#286]
```json
  {
  "tasks_version": 1.0,
  "project_name": "Pathways",
  "wbs_number": "P-004001.001",
  "perd_milestone": "Solution Sets 2020",
  "perd_output": "Solution Set #1",
  "status": "not started|waiting|active|completed",
  "planned start_date": "2020/04/01",
  "planned end_date": "2020/07/01",
  "actual_start_date": "not-started",
  "actual_end_date": "not-finished",
  "staff_resources": [
    { "name": "Kamel Haddad", 
      "estimated_work_days": 1, 
      "actual_work_days_to_date": 0
    },
    { "name": "Mike Lubun", 
      "estimated_work_days": 10, 
      "actual_work_days_to_date": 0
    },
    { "name": "Phylroy Lopez", 
      "estimated_work_days": 5, 
      "actual_work_days_to_date": 0
    }
  ],
  "procurement": [
    { "item": "description", "estimated_cost": 1, "actual_cost": 0},
    { "item": "description", "estimated cost": 1, "actual_cost": 0}
  ],
  "bi-weekly_update": "Enter comments here on current progress and raise any concerns or issues.",
  "waiting_on_task": "Url to task",
  "engineering_spec_completed": false,
  "engineering_code_implemented": false,
  "costing_spec_completed": true
[BTAP-Non RSMeans Costing Spec.docx](https://github.com/canmet-energy/btap_tasks/files/4773084/BTAP-Non.RSMeans.Costing.Spec.docx)

[BTAP-CCASHP Costing Example.docx](https://github.com/canmet-energy/btap_tasks/files/4773090/BTAP-CCASHP.Costing.Example.docx)

,
  "costing_code_implemented": false,
}
```

### Description
Write a decription of the issue. If it is a bug, please list all steps and attach a sample file, or sample ruby code that exhibits the bug. 

### Approach
After reviewing the issue, decribe your solution. If including git hyperlinks, ensure hyperlinks to existing code are to a version so it does not change with updates to git. 

### Testing Plan
Enter testing plan, if you are going to use an existing test or make a new one. 

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	
|CEO  |P-004001.001 |Cost-Effective Pathways for NZE, NZE Ready, and GHG Neutral Buildings |Phylroy Lopez
|GI     |P-002930.001 |Better Evidence For Better Building Codes |Alex Ferguson

**Comment by phylroy** [comment]
Mike added costing spec and example to task. 

[BTAP-Non RSMeans Costing Spec.docx](https://github.com/canmet-energy/btap_tasks/files/4773084/BTAP-Non.RSMeans.Costing.Spec.docx)

[BTAP-CCASHP Costing Example.docx](https://github.com/canmet-energy/btap_tasks/files/4773090/BTAP-CCASHP.Costing.Example.docx)

**Comment by phylroy** [comment]
completes and merged into btap-costing
